ansible_rulebook/engine.py

```diff
@@ -349,18 +349,20 @@ async def call_action(
                     else:
                         multi_match = rules_engine_result._m
                 variables_copy = variables.copy()
+                variables_eda = {}
+                variables_copy[KEY_EDA_VARS] = variables_eda
```

Comment:
@bzwei Does this impact playbooks written by our beta users, which might be using the root events/facts? If yes how do we communicate that.

Reply:
Yes any playbook references to {{ event.key }} now needs to change to {{ ansible_eda.event.key }}. This is a question to @josephpisciotta or @benthomasson. The good thing is now we have documented how to use such variables in playbooks.